### PR TITLE
Part 3 -User Interface

### PR DESCRIPTION
**Pull request by jt-is-coding**
Added the navbar and some styling, as well as fixing up some oversights in sample_data and removing scaffolding alerts

### CODE REVIEW

**Review by jb520 — APPROVED** [review]
Good job Jeremy!

File: config/routes.rb [hunk 1]
```diff
@@ -1,10 +1,14 @@
 Rails.application.routes.draw do
+  root "photos#index"
+
+  devise_for :users
```
Comment (YoungSong99) [inline config/routes.rb:4]: Nice! It increased a readability for routes.

File: lib/tasks/dev.rake [hunk 2]
```diff
@@ -15,7 +15,7 @@ task({ :sample_data => :environment }) do
   usernames << "Alice"
   usernames << "Bob"
 
-  username.each do |username|
+  usernames.each do |username|
```
Comment (YoungSong99) [inline lib/tasks/dev.rake:18]: great catch!

**Review by YoungSong99 — APPROVED** [review]
This code is clean and I like your commit message. It's really easy to follow!

**Review by rezahusain — APPROVED** [review]
Good job following convention Jeremy! Did everything you were supposed to from part 3. Keep up the good work.

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
```
Comment (JoseMDO) [inline app/controllers/application_controller.rb:2]: Nice!

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -6,11 +6,26 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render partial: "shared/cdn_assets" %>
```
Comment (JoseMDO) [inline app/views/layouts/application.html.erb:9]: Great job adding your CDN assets to a partial! This helps keep the layout file clean :)

File: app/views/shared/_navbar.html.erb [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" aria-expanded="false" href="#" role="button" data-bs-toggle="dropdown">Resources</a>
+          <div class="dropdown-menu">
+            <a class="dropdown-item" href="/photos">Photos</a>
+            <a class="dropdown-item" href="/comments">Comments</a>
```
Comment (JoseMDO) [inline app/views/shared/_navbar.html.erb:13]: Remember we can start using helper methods here like link_to :)

**Review by JoseMDO — APPROVED** [review]
Great job Jeremy!

**Pull request merged**